app/pages/admin/featured-project-toggle.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const FeaturedProjectToggle = ({ handleProjectChange, project }) => {
```

Comment:
You should use `function` instead of `const` to declare components. React uses `Function.name` to get the component display name, which doesn’t work with anonymous functions.